bucket/librewolf.json

```diff
@@ -4,7 +4,6 @@
     "homepage": "https://librewolf.net/",
     "license": "MPL-2.0",
     "url": "https://gitlab.com/librewolf-community/browser/windows/uploads/090a5145a8e3b3234f5c215ae26d61cb/librewolf-95.0.2.en-US.win64.zip",
-    "hash": "63a42ac89266004066efc412111bdc1f29a6de18df75e04faa61ccf45758c141",
```

Comment:
Hash is necessary.

Reply:
so does that mean I have to add hash to autoupdate?

Reply:
It's slightly tricky. Let me take a look.